portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,21 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Nitish Bharti's Portfolio</h1>
+      <p>
+          I'm Nitish Bharti persuing BTech Cse degree from IIT(ISM) Dhanbad.<br>
+          I like watching anime.<br>
+          tbh I'm just random type person who has no idea what's happening but I cope up with everything
+          and I do things deliberately and honestly with my heart and my energy.<br>
+          Focus and Patient these two things I follow<br><br>
+          <a href="https://www.linkedin.com/in/nitish-bharti/">LinkedIn Profile</a><br>
+          <a href="https://github.com/nitishbharti235">Github Profile</a> <br>
+      </p>
+      <p>This is my portfolio page and here is my faviourite anime character Ken kaneki.</p>
+      <img src="images/kaneki.png"/>
```

Comment:
`images/kaneki.png` - Be mindful of using images. Consider the personality, intent of use and the usage permissions on the image :)

Reply:
I checked it is free to use.

Reply:
Sounds good. marking this as resolved.